cfep-03.md

```diff
@@ -0,0 +1,51 @@
+
+<table>
+<tr><td> Title </td><td> Providing manually uploaded builds on the conda-forge anaconda channel</td>
+<tr><td> Status </td><td> Proposed </td></tr>
+<tr><td> Author(s) </td><td> Hadrien Mary &lt;hadrien.mary@gmail.com&gt;</td></tr>
+<tr><td> Created </td><td> Sept 27, 2016</td></tr>
+<tr><td> Updated </td><td> Sept 27, 2016</td></tr>
+<tr><td> Discussion </td><td> NA </td></tr>
+<tr><td> Implementation </td><td> NA </td></tr>
+</table>
+
+## Abstract
+
+Some packages are very hard to build on the currently used CI (circleci, Travis and AppVeyor). It's most of the time because of time limitations in the CI website and also because of the complexity of the build.
+
+An example is the Qt5 build that seems to be very hard to build (it tooks almost a year for Continuum folks to compile it for all platforms). See the Qt5 recipe build discussion ticket here https://github.com/conda-forge/qt-feedstock/issues/5.
+
+That being said it's excepted that a small amount of packages will be very hard to automatically build for various reasons. 
+
+Obviously we all want to keep this amount of packages small and have a maximun of automatically build packages on conda-forge.
+
+But a system/protocol to manually upload builds on conda-froge anaconda channel is necessary to circumvent some situations.
+
+## Specification
+
+A protocol to manually upload builds could look something like that :
+
+- Manually build the different packages for all targeted platforms.
+- Test for each platform on different version (multiple Linux distributions, Windows 7/10, etc).
+- Someone from the @conda-forge/core (having the permissions to upload on the conda channel) use conda command line to upload the package.
+- Build is available in the conda-forge anaconda channel.
```

Comment:
So one thought on this. Maybe so we can make it clear if something was manually built or not, would be to put this under a different label other than `main` like `manual`. We can still include this as a source at conda-forge, but it would be something one would need to add explicitly locally.

Reply:
The information is on the uploader username. No need for a new label.